src/openrct2-ui/windows/Options.cpp

```diff
@@ -426,8 +426,27 @@ namespace OpenRCT2::Ui::Windows
         window_options_misc_widgets,
         window_options_advanced_widgets,
     };
+
     // clang-format on
 
+    static constexpr std::pair<StringId, DrawingEngine> kDrawingEngineChoices[] = {
+        { STR_DRAWING_ENGINE_SOFTWARE_WITH_HARDWARE_DISPLAY, DrawingEngine::SoftwareWithHardwareDisplay },
```

Comment:
I would like to propose switching to `STR_DRAWING_ENGINE_SOFTWARE` as the name for the 'hardware-backed' software engine. The extra label was pretty confusing to begin with.

Reply:
I agree, lets just change the name for the frontend.